app/backend/db/crud.py

```diff
@@ -6,9 +6,23 @@
 from db import schemas
 
 # User:
+def create_user(db: Session, username: str, password: str) -> User:
+    new_user = User(english_name=username,
+                    username=username,
+                    password=password)
+    db.add(new_user)
+    db.commit()
+    db.refresh(new_user)
+    return new_user
+
 
 def get_user(db: Session, user_id: int) -> User:
-    return db.query(User).filter(User.id == user_id).one()
+    return db.query(User).filter(User.id == user_id).first()
```

Comment:
why?

Reply:
one() returns an exception if not exists.
i want to return None if not found

Reply:
and if two exists you are ok with it?

Reply:
It won't happen - it's a unique key.
Also in creation we are checking there isn't already one

Reply:
ok